src/submission/exceptions/exceptions.ts

```diff
@@ -321,14 +323,23 @@ export async function getExceptionManifestRecords(
 		createProgramExceptions(programId),
 	);
 
+	const sortedFollowUpExceptions = entityPropertyException
+		? entityPropertyException.follow_up.sort(sortExceptionRecordsBySubmitterId)
+		: [];
+
+	const sortedSpecimenExceptions = entityPropertyException
+		? entityPropertyException.specimen.sort(sortExceptionRecordsBySubmitterId)
+		: [];
+
+	const sortedTreatmentExceptions = entityPropertyException
+		? entityPropertyException.treatment.sort(sortExceptionRecordsBySubmitterId)
+		: [];
+
 	const entityPropertyExceptions: EntityPropertyExceptionRecord[] = entityPropertyException
-		? [
-				...entityPropertyException.specimen,
-				...entityPropertyException.treatment,
-				...entityPropertyException.follow_up,
-		  ]
+		? [...sortedFollowUpExceptions, ...sortedSpecimenExceptions, ...sortedTreatmentExceptions]
```

Comment:
already checked for the `entityPropertyException` above on line 336ish
at this point you'll have an array of some kind
this ternary is likely extraneous checking